build.sbt

```diff
@@ -1833,8 +1653,212 @@ lazy val runtime = (project in file("engine/runtime"))
   .dependsOn(`runtime-compiler`)
   .dependsOn(`connected-lock-manager`)
   .dependsOn(testkit % Test)
-  .dependsOn(`logging-service-logback` % "test->test")
-  .dependsOn(`runtime-test-instruments` % "test->compile")
+
+/** A project holding all the runtime integration tests. These tests require, among other things,
+  * the `org.enso.runtime` JPMS module, so it is easier to keep them in a separate project.
+  * For standard unit tests, use `runtime/Test`.
+  */
+lazy val `runtime-integration-tests` =
```

Comment:
Configuration of this project remains complicated, but at the end, they are _integration tests_ - they can be complicated.